src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -39,7 +39,7 @@
                                     {%- else -%}
                                             {{ redirect_route }}
                                     {%- endif %}
-                                    (<a href="{{ path('_profiler', { token: redirect.token }) }}">{{ redirect.token }}</a>)
+                                    (<a href="{{ path('_profiler', { token: redirect.token, panel: app.request.query.get('panel', 'request') }) }}">{{ redirect.token }}</a>)
```

Comment:
@javiereguiluz please use ``request``, not ``app.request``. The WebProfilerBundle does not depend on the ``app`` global variable before your change, making it compatible with Silex (where the ``app`` global variable is the Silex application itself, and so no compatible with ``app.request``)